### PR TITLE
Eng 1447 improved keycloak config

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 4f425a4088c8a301eb184534c57646c0ef35922a | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/1.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 1b4edf1f1f291d1bbe15ebab994aa6991778b000 | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/2.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 3af4dfd205169b63f506693a39b74b695e16b70a | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/3.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 3af4dfd205169b63f506693a39b74b695e16b70a | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/5.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 3af4dfd205169b63f506693a39b74b695e16b70a | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/6.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 83d73f63701918d00216f2cdcd2815894a5e9de1 | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-52/7.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr52**

File: src/main/bash/copy-images.sh
Comment (firegloves) [inline src/main/bash/copy-images.sh:17]: it seems to do nothing else than the echo. is it right?

File: src/main/java/org/entando/kubernetes/controller/coordinator/EntandoControllerCoordinator.java
Comment (firegloves) [inline src/main/java/org/entando/kubernetes/controller/coordinator/EntandoControllerCoordinator.java:61]: is this TODO to be solved in this ticket or is for the future?

File: src/main/bash/copy-images.sh
Comment (ampie) [reply src/main/bash/copy-images.sh:17]: Yeah Poste just wanted a list. I think I should revert this file to its original state....

File: src/main/java/org/entando/kubernetes/controller/coordinator/EntandoControllerCoordinator.java
Comment (ampie) [reply src/main/java/org/entando/kubernetes/controller/coordinator/EntandoControllerCoordinator.java:61]: It is some instructions for future improvements

**Review by firegloves — APPROVED** [review]
LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-controller-coordinator/pull/52#pullrequestreview-529949762" title="Approved">firegloves</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)~~ [firegloves]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
